exercises/baffling-birthdays/instructions.md

```diff
@@ -0,0 +1,15 @@
+# Instructions
+
+Your task is to implement a solution that verifies the Birthday Paradox's probabilities.
+
+To do this, you need to:
+
+- Determine if two birthdates have the same month and day.
+- Generate random birthdates.
+- Check if a set of randomly generated birthdates contains at least one matching pair.
+- Calculate the probability of at least one match for different group sizes.
+
+~~~~exercism/caution
+The Birthday Paradox assumes that birthdays are uniformly distributed within a single year.
+While the year should vary between different random birthdate generation calls, all birthdates within a single call must share the same year.
```

Comment:
We might also want to address leap years. The probability to be born on Feb 29 is 4(ish) time smaller than other dates, which changes the probability ever so slightly. I'd suggest adding that we can also assume that a year is 365 days.

Reply:
Good point.

Reply:
I've updated. Could you check if it is okay now?

Reply:
The "no leap year" line works, but I don't understand the point of "While the year should vary between different random birthdate generation calls, all birthdates within a single call _must_ share the same year.". Is this to further avoid leap year issues? If so, maybe it would work better to do filter out any Feb29s, or ask not generate them at all. Or we could leave it to the students to figure out how to avoid leap years.